test/conformance/kuttl/kuttl-test.yaml

```diff
@@ -9,6 +9,7 @@ testDirs:
 - ./test/conformance/kuttl/generate/clusterpolicy/standard/data/sync
 - ./test/conformance/kuttl/generate/clusterpolicy/standard/data/nosync
 - ./test/conformance/kuttl/generate/clusterpolicy/cornercases
+- ./test/conformance/kuttl/generate/namespacePolicy/standard/clone/sync/
```

Comment:
Ensure to update the path here according to an earlier comment.

Reply:
done.